modules/tribes/tests/client/components/TribesPage.component.tests.js

```diff
@@ -69,12 +40,20 @@ const renderAndWaitForTribes = async ({ user }) => {
 
 describe('TribesPage', () => {
 
+  const fake = generateFakeData({ users: 1, tribes: 5 });
```

Comment:
FYI We need to run this outside `beforeEach` because we need the data outside of `it`.

Here:
https://github.com/Trustroots/trustroots/blob/bd09f9102da595d5e41853be743e79b1e427f440/modules/tribes/tests/client/components/TribesPage.component.tests.js#L126